eos-phone-home

```diff
@@ -171,9 +172,9 @@ class PhoneHome(object):
             uploading_enabled = config['global'].getboolean('uploading_enabled',
                 uploading_enabled)
             environment = config['global'].get('environment', environment)
-        except Exception as e:
-            print("Unable to read %s, assuming fallback values." %
-                (permissions_file, ), e, file=sys.stderr)
+        except:
+            log.info("Unable to read %s, assuming fallback values.",
+                     permissions_file)
```

Comment:
Again I don't think the exception is useful:

```
INFO:__main__:Getting variable: metrics_enabled
INFO:__main__:Getting variable: metrics
INFO:__main__:Unable to read /etc/metrics/eos-metrics-permissions.conf, assuming fallback values.
Traceback (most recent call last):
  File "./eos-phone-home", line 171, in _get_metrics
    enabled = config['global'].getboolean('enabled', enabled)
  File "/usr/lib64/python3.6/configparser.py", line 959, in __getitem__
    raise KeyError(key)
KeyError: 'global'
INFO:__main__: - Metrics enabled: False
INFO:__main__:Getting variable: metrics_environment
INFO:__main__: - Metrics environment: unknown
```

Rather counter-intuitively, `ConfigParser.read` does not throw if the path doesn't exist:

```
In [5]: p.read?
Signature: p.read(filenames, encoding=None)
Docstring:
Read and parse a filename or a list of filenames.

Files that cannot be opened are silently ignored; this is
designed so that you can specify a list of potential
configuration file locations (e.g. current directory, user's
home directory, systemwide directory), and all existing
configuration files in the list will be read.  A single
filename may also be given.

Return list of successfully read files.
```

Perhaps a better way to write this function would be:

```
config = configparser.ConfigParser()
if not config.read(permissions_file):
    log.info("couldn't read %s" ...)
enabled = config.getboolean('global', 'enabled', False)
...
```

I'll cook up a patch.